### PR TITLE
Add Openshift instruction to External Secrets Operator

### PR DESCRIPTION
**Pull request by rszumlakowski**
[TANZUSC-5098]

Please apply this documentation to TAP 1.7.0 and up.

### CODE REVIEW

File: external-secrets/install-external-secrets-operator.hbs.md [hunk 1]
```diff
@@ -65,11 +70,55 @@ To install External Secrets Operator:
...
+      Forbidden: not usable by user or serviceaccount, provider "hostnetwork-v2":
+      Forbidden: not usable by user or serviceaccount, provider "hostnetwork": Forbidden:
+      not usable by user or serviceaccount, provider "hostaccess": Forbidden: not
+      usable by user or serviceaccount, provider "node-exporter": Forbidden: not usable
+      by user or serviceaccount, provider "privileged": Forbidden: not usable by user
+      or serviceaccount]'
+
+   You can work around this issue by installing the `full` TAP profile and then installing the
+   External Secrets Operator package.
+
```
Comment (rashedkvm) [inline external-secrets/install-external-secrets-operator.hbs.md:117]:
```suggestion

```
Suggestion: Please remove this section of the text. External Secrets Operator is a pre-req for TAP gitops install.

File: external-secrets/install-external-secrets-operator.hbs.md [hunk 2]
```diff
@@ -36,17 +36,22 @@ To install External Secrets Operator:
    tanzu package install external-secrets \
      --package external-secrets.apps.tanzu.vmware.com \
      --version VERSION-NUMBER \
+     --values-file VALUES_FILE.YAML \
```
Comment (rashedkvm) [inline external-secrets/install-external-secrets-operator.hbs.md:39]: ` --values-file VALUES_FILE.YAML` - This use of this file is optional.

File: external-secrets/install-external-secrets-operator.hbs.md [hunk 3]
```diff
@@ -65,11 +70,55 @@ To install External Secrets Operator:
...
+   Operator so that OpenShift can replace the security settings with its own.
+
+   If you are running the External Secrets Operator on any other platforms than
+   you can leave `kubernetes_distribution` as an empty string (i.e.: `""`) or
+   omit the value entirely.
+
+   The External Secrets Operator should be able to inherit this setting from the
+   `shared.kubernetes_distribution` in the top-level values file applied to the
+   TAP install itself.
+
```
Comment (rashedkvm) [inline external-secrets/install-external-secrets-operator.hbs.md:91]:
```suggestion

```
Suggestion: Please remove this section of the text. External Secrets Operator is a pre-req for TAP gitops install.

File: external-secrets/install-external-secrets-operator.hbs.md (hunk 2 above)
Comment (rszumlakowski) [inline external-secrets/install-external-secrets-operator.hbs.md:39]:
```suggestion
     --values-file VALUES_FILE.YAML \ # The use of this file is optional
```

**Review by rashedkvm — APPROVED** [review]
Lgtm

**Pull request merged**